src/index.d.ts

```diff
@@ -102,6 +102,8 @@ interface SendBirdStateConfig {
     resizingWidth?: number | string,
     resizingHeight?: number | string,
   };
+  isTypingIndicatorEnabledOnChannelList?: boolean;
+  isMessageReceiptStatusEnabledOnChannelList?: boolean;
```

Comment:
Add these to props for SendBirdProvider, App, 
Also, please add all the other prop changes here too